packages/azure_frontdoor/data_stream/access/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,184 @@
+---
+description: Pipeline for processing azure frontdoor access logs
+processors:
+  - set:
+      field: ecs.version
+      value: "8.5.0"
+  - set:
+      field: event.category
+      value: [network]
+  - set:
+      field: event.type
+      value: [connection]
+  - rename:
+      field: message
+      target_field: event.original
+      ignore_missing: true
+  - set:
+      field: cloud.provider
+      value: azure
+  - drop:
+      description: Drop if  inavlid json
+      if: 'ctx?.event.original.contains(''"records"'')'
```

```suggestion
      if: 'ctx.event?.original?.contains(''"records"'')'
```
Since `ctx` is never null

---

updated to `if: 'ctx.event?.original != null && ctx.event.original.contains(''"records"'')'`